src/core/utils/helpers/list-mapper.ts

```diff
@@ -36,6 +36,24 @@ function getYearFromDataString(date: string) {
   return new Date(date).getFullYear();
 }
 
+function getSeriesString(
```

Comment:
We also use series in our part of the codebase. Could you please double check if we do it in the same manner?